group_vars/sn07.yml

```diff
@@ -0,0 +1,594 @@
+---
```

Comment:
Is this file only temporary here or do we really plan to keep two duplicated files?

Reply:
Sorry, this should not be part of this PR, it's a mistake. I will remove this commit shortly.

There are some changes required related to RHEL9. We are also checking individual roles and also updating them to work for RHEL 9. So we might end up with a dedicated sn07 playbook and a corresponding vars file.

Reply:
And I think we can use tat plabook for sn06, after we upgraded that machine to rocky linux 9.
Maybe we should rename the playbook later to "headnode" or something like this

Reply:
Yes, that would be my ideal.